code/modules/mob/living/carbon/brain/robotic_brain.dm

```diff
@@ -90,10 +90,13 @@
 
 /obj/item/mmi/robotic_brain/transfer_identity(mob/living/carbon/H)
 	name = "[src] ([H])"
-	if(isnull(brainmob.dna))
-		brainmob.dna = H.dna.Clone()
-	brainmob.name = brainmob.dna.real_name
-	brainmob.real_name = brainmob.name
+
+	brainmob.dna = H.dna.Clone()
+	// I'm not sure we can remove species override. There might be some loophole
+	// that would allow posibrains to be cloned without this.
+	brainmob.dna.species = new /datum/species/machine()
```

Comment:
Does this mean the mob in the MMI will get the change monitor action button, like IPC mobs get?
`var/datum/action/innate/change_monitor/monitor`

Reply:
It does not
![dreamseeker-20201127-171514](https://user-images.githubusercontent.com/7831163/100472563-6031ab80-30d4-11eb-9de1-95770d4326b1.png)

(ignore the fact that the research scanner action is duplicated - this is the same in live version)

Reply:
Ah right because its set directly and not going through `.set_species()`.